python/tests/test_topology.py

```diff
@@ -98,11 +98,11 @@ def generate_segments(n, sequence_length=100, seed=None):
     return segs
 
 
-def kc_distance_tree(tree1, tree2, topo_v_age=0):
+def kc_distance_tree(tree1, tree2, lambda_param=0):
```

Comment:
``lambda_param`` is awkward. The convention is usually to call it ``lambda_`` to avoid clashes with built-in symbols.

Reply:
Got it, thanks for checking this out!